.github/workflows/test_examples.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest]
-        toolchain: [stable-2020-10-08]
+        toolchain: [stable]
```

Comment:
thank you, why did we even have the other version. didn't notice 👍🏼

Reply:
idk it was the version used across the board before, I assume for consistency, but is counter-intuitive for a lib like the SDK